src/pages/experiments/index.js

```diff
@@ -106,18 +113,29 @@ let Experiment = ({
             <h1 className="experiment__header-title mobile-p">
               <HText>{experiment.title || 'No Title.'}</HText>
             </h1>
-            <div>
+            <div className="flex-row experiment__dataset-buttons">
               {numDownloadableSamples === 0 ? (
                 <RequestExperimentButton accessionCode={accessionCode} />
               ) : (
-                <DataSetSampleActions
-                  dataSetSlice={{
-                    [experiment.accession_code]: {
-                      all: true,
-                      total: numDownloadableSamples,
-                    },
-                  }}
-                />
+                <>
+                  <DataSetSampleActions
+                    dataSetSlice={{
+                      [experiment.accession_code]: {
+                        all: true,
+                        total: numDownloadableSamples,
+                      },
+                    }}
+                  />
+                  {!dataset.is_processing ? (
+                    <DownloadExperiment
+                      experiment={experiment}
+                      dataset={dataset}
+                      setDataset={setDataset}
```

Comment:
* The experiment page wants to be able to manage the state (in order to update multiple instances of the same dataset being downloadable) 
* I think its better if we move that management outside of the component, dataset and setDataset can also be wrapped this way
* This configuration is just one, probably one of the more complex examples, but in the future other components won't be so deeply nested (ie) dataset download page with will just need to have the dataset its configuring options for and a way to update it because of the way the API works, ideally we will move away from that in the future and can treat it more restfully. 📅